README.md

```diff
@@ -1,2 +1,60 @@
 # JS-To-Do-List
 Learning the concepts of Single Responsibility and Tightly Coupled Objects in Javascript
+
+## Built With
+
+- HTML,
+- CSS,
+- JavaScript,
+- Bootstrap
+- Webpack
+
+## Live demo Link
+
+[live link](https://rawcdn.githack.com/OdongAlican/JS-To-Do-List/3a86b65dc66bbd7be54e098d4dd2f6db1ce57300/dist/index.html)
+
+![Landing page](1.png)
+![To-Do page](2.png)
+![Add To-Do Form](3.png)
```

Comment:
Files `1.png`, `2.png`, and `3.png` are not in the root directory of your project. We need to specify the correct directory or they will not be displayed in the README file. Change this to:
```
(images/1.png)
(images/2.png)
(images/3.png)
```